### PR TITLE
Refactor kernel-picker to remove atom-space-pen-views

### PR DESCRIPTION
**Pull request by BenRussert**
This pr is heavily influenced by [this example.](https://github.com/atom/grammar-selector/blob/master/lib/grammar-list-view.js)

I haven't found any bugs yet, but this probably needs a little testing.

### CODE REVIEW

File: lib/kernel-picker.js [hunk 1]
```diff
@@ -1,70 +1,64 @@
...
-import { SelectListView } from "atom-space-pen-views";
+import SelectListView from "atom-select-list";
 import _ from "lodash";
 
 import { log } from "./utils";
 
-// View to display a list of grammars to apply to the current editor.
-export default class SignalListView extends SelectListView {
-  initialize(kernelSpecs) {
+export default class SignalListView {
```
Comment (BenRussert) [inline lib/kernel-picker.js]: I guess we could call this class `KernelPicker` right? Left as is to minimize diff for now.

File: lib/kernel-picker.js [hunk 2]
```diff
@@ -1,70 +1,64 @@
...
+        element.textContent = item.display_name;
+        return element;
+      },
+      didConfirmSelection: item => {
+        log("Selected kernel:", item);
+        if (this.onConfirmed) this.onConfirmed(item);
+        this.cancel();
+      },
+      didCancelSelection: () => this.cancel(),
+      emptyMessage: "No running kernels found."
```
Comment (n-riesco) [inline lib/kernel-picker.js]: I've just noticed that this message should've been `"No kernels found"`.

**Comment by n-riesco** [comment]
I've tested it with multiple kernels, multi-language files. And both, the functionality to start a kernel and the status bar switcher work for me.

File: lib/kernel-picker.js (hunk 1 above)
Comment (lgeiger) [inline lib/kernel-picker.js]: 👍 `KernelPicker` is better

File: lib/kernel-picker.js (hunk 2 above)
Comment (lgeiger) [inline lib/kernel-picker.js]: Good catch!

**Pull request merged**